docs/schemas/raw/lambda_config.yaml

```diff
@@ -93,6 +110,8 @@ required:
 - table
 - config_table
 - config_cache_ttl
+- connect_timeout
```

Comment:
Note that adding these as "required" means it won't be backwards-compatible with existing config files so it will force an update in the playbooks at the same time.  You might want to consider not adding them to required, or only adding them to required later.

Reply:
Yeah, that's why I've made this dependant on the cdn-playbooks change. Once that's fully deployed this will be safe.